05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT_IFILE" not found; SQL statement:
alter table SNAPSHOT_IFILE drop constraint SF_S_ID_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT_IFILE" not found; SQL statement:
alter table SNAPSHOT_IFILE drop constraint SF_F_ID_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table SNAPSHOT drop constraint S_CREATION_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER" not found; SQL statement:
alter table CONTAINER drop constraint C_LOCK_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER" not found; SQL statement:
alter table CONTAINER drop constraint C_CREATION_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER" not found; SQL statement:
alter table CONTAINER drop constraint C_MODIFICATION_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER_SNAPSHOT" not found; SQL statement:
alter table CONTAINER_SNAPSHOT drop constraint CS_C_ID_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER_SNAPSHOT" not found; SQL statement:
alter table CONTAINER_SNAPSHOT drop constraint CS_S_ID_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY" not found; SQL statement:
alter table DIRECTORY drop constraint D_PARENT_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY" not found; SQL statement:
alter table DIRECTORY drop constraint D_CREATION_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY" not found; SQL statement:
alter table DIRECTORY drop constraint D_MODIFICATION_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY" not found; SQL statement:
alter table DIRECTORY drop constraint D_DELETED_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY_CONTAINER" not found; SQL statement:
alter table DIRECTORY_CONTAINER drop constraint DC_D_ID [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY_CONTAINER" not found; SQL statement:
alter table DIRECTORY_CONTAINER drop constraint DC_C_ID [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY_CONTAINER" not found; SQL statement:
alter table DIRECTORY_CONTAINER drop constraint DC_DELETED_USER_FK [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY_CONTAINER" not found; SQL statement:
drop table DIRECTORY_CONTAINER [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "DIRECTORY" not found; SQL statement:
drop table DIRECTORY [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER_SNAPSHOT" not found; SQL statement:
drop table CONTAINER_SNAPSHOT [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONTAINER" not found; SQL statement:
drop table CONTAINER [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT_IFILE" not found; SQL statement:
drop table SNAPSHOT_IFILE [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
drop table SNAPSHOT [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "IFILE" not found; SQL statement:
drop table IFILE [42102-154]
05-26 21:15:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SUSER" not found; SQL statement:
drop table SUSER [42102-154]
05-26 21:15:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2732)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2824)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1075)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mouseReleased(MouseUtils.java:184)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:273)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:272)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:272)
	at java.awt.Component.processMouseEvent(Component.java:6289)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3267)
	at java.awt.Component.processEvent(Component.java:6054)
	at java.awt.Container.processEvent(Container.java:2041)
	at java.awt.Component.dispatchEventImpl(Component.java:4652)
	at java.awt.Container.dispatchEventImpl(Container.java:2099)
	at java.awt.Component.dispatchEvent(Component.java:4482)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4577)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4238)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4168)
	at java.awt.Container.dispatchEventImpl(Container.java:2085)
	at java.awt.Window.dispatchEventImpl(Window.java:2478)
	at java.awt.Component.dispatchEvent(Component.java:4482)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:644)
	at java.awt.EventQueue.access$000(EventQueue.java:85)
	at java.awt.EventQueue$1.run(EventQueue.java:603)
	at java.awt.EventQueue$1.run(EventQueue.java:601)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
	at java.awt.EventQueue$2.run(EventQueue.java:617)
	at java.awt.EventQueue$2.run(EventQueue.java:615)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:614)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:148)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
05-26 21:15:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:29 jdbc[2]: The connection was not closed by the application and is garbage collected [90018-154]
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:98)
	at org.h2.Driver.connect(Driver.java:72)
	at org.netbeans.modules.db.explorer.DbDriverManager.getConnection(DbDriverManager.java:131)
	at org.netbeans.modules.db.explorer.DatabaseConnection.doConnect(DatabaseConnection.java:781)
	at org.netbeans.modules.db.explorer.DatabaseConnection.access$100(DatabaseConnection.java:111)
	at org.netbeans.modules.db.explorer.DatabaseConnection$2.run(DatabaseConnection.java:847)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
05-26 21:15:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:41:08 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:41:08 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:41:08 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:46:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:46:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 09:46:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 11:52:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT_MODULE" not found; SQL statement:
alter table SNAPSHOT_MODULE drop constraint SM_S_ID_FK [42102-154]
06-06 11:52:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT_MODULE" not found; SQL statement:
drop table SNAPSHOT_MODULE [42102-154]
06-06 11:52:28 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 11:52:28 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 11:52:28 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 21:20:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 21:20:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-06 21:20:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 15:21:02 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 15:21:02 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 15:21:02 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 18:29:10 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 18:29:10 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 18:29:10 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 20:50:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 20:50:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-07 20:50:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-09 18:13:01 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-09 18:13:01 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-09 18:13:01 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-10 15:04:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:222)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-10 15:04:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-10 15:04:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:189)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:106)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-29 10:43:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-29 10:43:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
06-29 10:43:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-02 17:15:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-02 17:15:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-02 17:15:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 12:30:59 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 12:30:59 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 12:30:59 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE SNAPSHOT
(
        S_ID VARCHAR(36) NOT NULL,
        S_CREATION_DATE TIMESTAMP NOT NULL,
        S_CREATION_USER VARCHAR(36) NOT NULL,
        S_TRANSFERED INT NOT NULL
        PRIMARY KEY ([*]S_ID)
) "; expected ")"; SQL statement:
create table SNAPSHOT
(
        S_ID varchar(36) not null,
        S_CREATION_DATE timestamp not null,
        S_CREATION_USER varchar(36) not null,
        S_TRANSFERED int not null
        primary key (S_ID)
) [42001-154]
07-26 13:59:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table SNAPSHOT add constraint S_CREATION_USER_FK foreign key (S_CREATION_USER) references SUSER(U_ID) [42102-154]
07-26 13:59:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table SNAPSHOT_IFILE add constraint SF_S_ID_FK foreign key (SF_S_ID) references SNAPSHOT(S_ID) [42102-154]
07-26 13:59:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table CONTAINER_SNAPSHOT add constraint CS_S_ID_FK foreign key (CS_S_ID) references SNAPSHOT(S_ID) [42102-154]
07-26 13:59:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table SNAPSHOT_MODULE add constraint SM_S_ID_FK foreign key (SM_S_ID) references SNAPSHOT(S_ID) [42102-154]
07-26 13:59:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Constraint "SF_S_ID_FK" not found; SQL statement:
alter table SNAPSHOT_IFILE drop constraint SF_S_ID_FK [90057-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:40)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.executeSQLStatement(SQLExecutionHelper.java:604)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.initialDataLoad(SQLExecutionHelper.java:113)
	at org.netbeans.modules.db.dataview.output.DataView.create(DataView.java:105)
	at org.netbeans.modules.db.dataview.api.DataView.create(DataView.java:74)
	at org.netbeans.modules.db.sql.execute.SQLExecuteHelper.execute(SQLExecuteHelper.java:119)
	at org.netbeans.modules.db.sql.loader.SQLEditorSupport$SQLExecutor.run(SQLEditorSupport.java:567)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
alter table SNAPSHOT drop constraint S_CREATION_USER_FK [42102-154]
07-26 13:59:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Constraint "CS_S_ID_FK" not found; SQL statement:
alter table CONTAINER_SNAPSHOT drop constraint CS_S_ID_FK [90057-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:40)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.executeSQLStatement(SQLExecutionHelper.java:604)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.initialDataLoad(SQLExecutionHelper.java:113)
	at org.netbeans.modules.db.dataview.output.DataView.create(DataView.java:105)
	at org.netbeans.modules.db.dataview.api.DataView.create(DataView.java:74)
	at org.netbeans.modules.db.sql.execute.SQLExecuteHelper.execute(SQLExecuteHelper.java:119)
	at org.netbeans.modules.db.sql.loader.SQLEditorSupport$SQLExecutor.run(SQLEditorSupport.java:567)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Constraint "SM_S_ID_FK" not found; SQL statement:
alter table SNAPSHOT_MODULE drop constraint SM_S_ID_FK [90057-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:40)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.executeSQLStatement(SQLExecutionHelper.java:604)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.initialDataLoad(SQLExecutionHelper.java:113)
	at org.netbeans.modules.db.dataview.output.DataView.create(DataView.java:105)
	at org.netbeans.modules.db.dataview.api.DataView.create(DataView.java:74)
	at org.netbeans.modules.db.sql.execute.SQLExecuteHelper.execute(SQLExecuteHelper.java:119)
	at org.netbeans.modules.db.sql.loader.SQLEditorSupport$SQLExecutor.run(SQLEditorSupport.java:567)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOT" not found; SQL statement:
drop table SNAPSHOT cascade constraints [42102-154]
07-26 13:59:44 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:44 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-26 13:59:44 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-27 08:42:21 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-27 08:42:21 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
07-27 08:42:21 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:48:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "IFILENEW" not found; SQL statement:
alter table IFILENEW drop constraint FN_ID_FK [42102-154]
08-28 18:48:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOTNEW" not found; SQL statement:
alter table SNAPSHOTNEW drop constraint SN_ID_FK [42102-154]
08-28 18:48:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SNAPSHOTNEW" not found; SQL statement:
drop table SNAPSHOTNEW cascade constraints [42102-154]
08-28 18:48:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "IFILENEW" not found; SQL statement:
drop table IFILENEW cascade constraints [42102-154]
08-28 18:48:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "I_FILE" not found; SQL statement:
alter table IFILENEW add constraint FN_ID_FK foreign key (FN_ID) references I_FILE(F_ID) [42102-154]
08-28 18:48:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FN_ID_FK" not found; SQL statement:
alter table IFILENEW drop constraint FN_ID_FK [90057-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:40)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.executeSQLStatement(SQLExecutionHelper.java:604)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.initialDataLoad(SQLExecutionHelper.java:113)
	at org.netbeans.modules.db.dataview.output.DataView.create(DataView.java:105)
	at org.netbeans.modules.db.dataview.api.DataView.create(DataView.java:74)
	at org.netbeans.modules.db.sql.execute.SQLExecuteHelper.execute(SQLExecuteHelper.java:119)
	at org.netbeans.modules.db.sql.loader.SQLEditorSupport$SQLExecutor.run(SQLEditorSupport.java:567)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:48:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "I_FILE" not found; SQL statement:
alter table IFILENEW add constraint FN_ID_FK foreign key (FN_ID) references I_FILE(F_ID) [42102-154]
08-28 18:49:08 jdbc[2]: The connection was not closed by the application and is garbage collected [90018-154]
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:98)
	at org.h2.Driver.connect(Driver.java:72)
	at org.netbeans.modules.db.explorer.DbDriverManager.getConnection(DbDriverManager.java:131)
	at org.netbeans.modules.db.explorer.DatabaseConnection.doConnect(DatabaseConnection.java:781)
	at org.netbeans.modules.db.explorer.DatabaseConnection.access$100(DatabaseConnection.java:111)
	at org.netbeans.modules.db.explorer.DatabaseConnection$2.run(DatabaseConnection.java:847)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FN_ID_FK" not found; SQL statement:
alter table IFILENEW drop constraint FN_ID_FK [90057-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:40)
	at org.h2.command.CommandContainer.update(CommandContainer.java:69)
	at org.h2.command.Command.executeUpdate(Command.java:212)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.executeSQLStatement(SQLExecutionHelper.java:604)
	at org.netbeans.modules.db.dataview.output.SQLExecutionHelper.initialDataLoad(SQLExecutionHelper.java:113)
	at org.netbeans.modules.db.dataview.output.DataView.create(DataView.java:105)
	at org.netbeans.modules.db.dataview.api.DataView.create(DataView.java:74)
	at org.netbeans.modules.db.sql.execute.SQLExecuteHelper.execute(SQLExecuteHelper.java:119)
	at org.netbeans.modules.db.sql.loader.SQLEditorSupport$SQLExecutor.run(SQLEditorSupport.java:567)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:49:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:49:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 18:49:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 22:16:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 22:16:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
08-28 22:16:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 15:11:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 15:11:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 15:11:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 19:50:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 19:50:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-06 19:50:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-13 11:44:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-13 11:44:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-13 11:44:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-14 16:05:06 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-14 16:05:06 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
09-14 16:05:06 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
10-02 13:13:35 jdbc[2]: The connection was not closed by the application and is garbage collected [90018-154]
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:98)
	at org.h2.Driver.connect(Driver.java:72)
	at org.netbeans.modules.db.explorer.DbDriverManager.getConnection(DbDriverManager.java:131)
	at org.netbeans.modules.db.explorer.DatabaseConnection.doConnect(DatabaseConnection.java:781)
	at org.netbeans.modules.db.explorer.DatabaseConnection.access$100(DatabaseConnection.java:111)
	at org.netbeans.modules.db.explorer.DatabaseConnection$2.run(DatabaseConnection.java:847)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
10-03 18:43:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
10-03 18:43:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
10-03 18:43:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b0-ed1e-11e0-be50-0800200c9a66','Administrateur OPHELIE Gestion (OPGA)','OPG.ADMINISTRATEURS') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b1-ed1e-11e0-be50-0800200c9a66','Utilisateur OPHELIE Gestion (OPGB)','OPG.UTILISATEURS') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b2-ed1e-11e0-be50-0800200c9a66','Administrateur BSX OPHELIE Gestion (OPGC)','OPG.ADMIN_BSX') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b3-ed1e-11e0-be50-0800200c9a66','Administrateur Noyau OPHELIE Gestion (OPGD)','OPG.ADMIN_NOYAU') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b4-ed1e-11e0-be50-0800200c9a66','Administrateur List APP OPHELIE Gestion (OPGE)','OPG.ADMIN_LISTE_APP') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b5-ed1e-11e0-be50-0800200c9a66','Administrateur OPHELIE Distrib (OPDA)','OPDA.ADMINISTRATEURS') [42102-154]
10-05 01:24:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SACCESS" not found; SQL statement:
insert into SACCESS values ('59abd4b6-ed1e-11e0-be50-0800200c9a66','Utilisateur OPHELIE Distrib (OPDB)','OPDA.UTILISATEURS') [42102-154]
11-07 02:21:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.action.ExecuteCommandAction.enable(ExecuteCommandAction.java:66)
	at org.openide.util.actions.NodeAction$DelegateAction.resultChanged(NodeAction.java:615)
	at org.openide.util.actions.NodeAction$DelegateAction.<init>(NodeAction.java:560)
	at org.openide.util.actions.NodeAction.createContextAwareInstance(NodeAction.java:357)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2732)
	at org.openide.util.Utilities.actionsToPopup(Utilities.java:2824)
	at org.openide.explorer.view.TreeView.createPopup(TreeView.java:1075)
	at org.openide.explorer.view.TreeView$PopupAdapter.showPopup(TreeView.java:1558)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mouseReleased(MouseUtils.java:184)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:273)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:272)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:272)
	at java.awt.Component.processMouseEvent(Component.java:6289)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3267)
	at java.awt.Component.processEvent(Component.java:6054)
	at java.awt.Container.processEvent(Container.java:2041)
	at java.awt.Component.dispatchEventImpl(Component.java:4652)
	at java.awt.Container.dispatchEventImpl(Container.java:2099)
	at java.awt.Component.dispatchEvent(Component.java:4482)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4577)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4238)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4168)
	at java.awt.Container.dispatchEventImpl(Container.java:2085)
	at java.awt.Window.dispatchEventImpl(Window.java:2478)
	at java.awt.Component.dispatchEvent(Component.java:4482)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:644)
	at java.awt.EventQueue.access$000(EventQueue.java:85)
	at java.awt.EventQueue$1.run(EventQueue.java:603)
	at java.awt.EventQueue$1.run(EventQueue.java:601)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
	at java.awt.EventQueue$2.run(EventQueue.java:617)
	at java.awt.EventQueue$2.run(EventQueue.java:615)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:614)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:148)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
11-07 02:21:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:21:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:21:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:21:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:21:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:21:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:40:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:40:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-07 02:40:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-14 02:08:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.removeAllNodes(NodeProvider.java:227)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.removeAllNodes(NodeRegistry.java:154)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:312)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-14 02:08:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode.refresh(BaseNode.java:181)
	at org.netbeans.modules.db.explorer.node.ConnectionNode$2.run(ConnectionNode.java:313)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
11-14 02:08:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-154]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1374)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1352)
	at org.h2.jdbc.JdbcConnection.getWarnings(JdbcConnection.java:568)
	at org.netbeans.modules.db.explorer.DatabaseConnection.isVitalConnection(DatabaseConnection.java:299)
	at org.netbeans.modules.db.explorer.node.ConnectionNode.getIconBase(ConnectionNode.java:379)
	at org.netbeans.api.db.explorer.node.BaseNode.updateProperties(BaseNode.java:264)
	at org.netbeans.api.db.explorer.node.BaseNode.update(BaseNode.java:251)
	at org.netbeans.api.db.explorer.node.BaseNode$1.stateChanged(BaseNode.java:201)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.stateChanged(NodeRegistry.java:182)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:133)
	at org.openide.util.ChangeSupport.fireChange(ChangeSupport.java:119)
	at org.netbeans.api.db.explorer.node.NodeProvider.setNodes(NodeProvider.java:194)
	at org.netbeans.modules.db.explorer.node.CatalogNodeProvider.initialize(CatalogNodeProvider.java:141)
	at org.netbeans.api.db.explorer.node.NodeProvider.getNodes(NodeProvider.java:111)
	at org.netbeans.modules.db.explorer.node.NodeRegistry.getNodes(NodeRegistry.java:167)
	at org.netbeans.api.db.explorer.node.ChildNodeFactory.createKeys(ChildNodeFactory.java:107)
	at org.openide.nodes.AsynchChildren.run(AsynchChildren.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1424)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:1968)
